src/ability.rs

```diff
@@ -166,7 +167,9 @@ impl AbilityScore {
     /// To determine an ability modifier without consulting the table, subtract 10 from
     /// the ability score and then divide the total by 2 (round down).
     pub fn modifier(&self) -> i8 {
-        ((self.value as i8 + 10) / 2) - 10
+        (((i16::from(self.value) + 10) / 2) - 10)
```

Comment:
I somehow don't like but cannot think of a nice alternative

Reply:
Made up formula, but it works